vpd-tool/include/vpd_tool.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <string>
+
+namespace vpd
+{
+/**
+ * @brief Class to support operations on VPD.
+ *
+ * The class provides API's to,
+ * Read keyword value from DBus/hardware.
+ * Update keyword value to DBus/hardware.
+ * Dump DBus object's critical information.
+ * Fix system VPD if any mismatch between DBus and hardware data.
+ * Reset specific system VPD keywords to its default value.
+ * Force VPD collection for hardware.
+ */
+class VpdTool
+{
+  public:
+    /**
+     * @brief Read keyword value.
+     *
+     * API to read VPD keyword's value from the given input path.
+     * If the provided i_onHardware option is true, read keyword's value from
+     * the hardware. Otherwise read keyword's value from DBus.
+     *
+     * Note: In case read keyword's value from DBus, the return value from DBus
```

Comment:
are the outputs of both hardware read and dbus read saved to file? why do we have this Note:?

Reply:
This note is nothing to do with the saving to file or not.

I Added this not from previous comment history. 
I will remove this note as only keeping about read from DBus doesn't adds any additional value.